.github/workflows/check_examples.yaml

```diff
@@ -1,22 +1,30 @@
-name: "check examples"
+name: Check examples
 on:
   push:
-    branches:
-    - master
-    - dev
+    paths:
+    - 'examples/**'
```

Comment:
We still want to check the examples even if they haven't changed. A code change could break one of the examples and it might not be covered by the acceptance tests - definitely not at the moment.

Reply:
good catch, thanks!